app/src/main/res/values-en-rGB/strings.xml

```diff
@@ -0,0 +1,23 @@
+<resources>
+
+    <string name="title_favourites">Favourites</string>
+    <string name="notification_favourite_format">%s favourited your toot</string>
+    <string name="action_favourite">Favourite</string>
+    <string name="action_unfavourite">Remove favourite</string>
+    <string name="action_view_favourites">Favourites</string>
+    <string name="action_open_faved_by">Show favourites</string>
+    <string name="pref_title_notification_filter_favourites">my posts are favourited</string>
+    <string name="notification_favourite_name">Favourites</string>
+    <string name="notification_favourite_description">Notifications when your toots get marked as favourite</string>
+
+    <string name="description_status_favourited">Favourited</string>
+
+    <string name="error_authorization_unknown">An unidentified authorisation error occurred.</string>
+    <string name="error_authorization_denied">Authorisation was denied.</string>
```

Comment:
```suggestion
    <string name="error_authorization_denied">Authorisation was denied.</string>
    <string name="title_licenses">Licences</string>
    
    <string name="about_tusky_license">Tusky is free and open-source software.
        It is licensed under the GNU General Public License Version 3.
You can view the licence here: https://www.gnu.org/licenses/gpl-3.0.en.html</string>
    <string name="report_description_remote_instance">The account is from another server. Send an anonymised copy of the report there as well?</string>
```

Reply:
(I didn't touch `GNU General Public License` nor `Apache License` since they're the proper names of the licenses)